pyscriptjs/package.json

```diff
@@ -12,8 +12,8 @@
     "lint": "eslint './src/**/*.{js,svelte,html,ts}'",
     "lint:fix": "eslint --fix './src/**/*.{js,svelte,html,ts}'",
     "xprelint": "npm run format",
-    "test": "jest --coverage",
-    "test:watch": "jest --watch"
+    "test": "cross-env NODE_OPTIONS=--experimental-vm-modules jest --coverage",
```

Comment:
same here... (it's just good to socialize why we **need** to run with cros-env)

Reply:
JSON files cannot have comments unfortunately